jekyll/_cci2/collect-test-data.md

```diff
@@ -1,47 +1,63 @@
 
 * TOC
 {:toc}
 
+## Overview
+
+Using the **`store_test_results` step** gives you access to:
+
+* The **Tests** pane in the CircleCI web app, as described in the {{site.url}}/{{site.baseurl}}/2.0/collect-test-data/[Collecting Test Metadata guide].
+* Test insights and flaky test detection.
+* Test splitting
+
+Alternatively, storing test results as **artifacts** means you can look at the raw xml. This can be useful when debugging issues with setting up your project's test results handling, for example, working out if you are uploading incorrect files. To see test results as build artifacts, upload them using the [`store_artifacts`]({{ site.baseurl}}/2.0/configuration-reference/#store_artifacts) step.
+
+## Using the `store_test_results` step
+
 Using the [`store_test_results`]({{ site.baseurl}}/2.0/configuration-reference/#store_test_results) step allows you to
-not only upload and store test results, but also provides an easy-to-read UI of your passing/failing tests in the CircleCI
-application.
+not only upload and store test results, but also provides a view of your passing/failing tests in the CircleCI
+web app.
 
-You can access the test results interface from the *Tests* tab when viewing any particular [job]({{ site.baseurl}}/2.0/concepts/#jobs),
-as seen below.
+You can access the test results from the **Tests** tab when viewing a job, as shown below.
 
 ![store-test-results-view]( {{ site.baseurl }}/assets/img/docs/test-summary.png)
 
-To see test results as build artifacts, upload them using the [`store_artifacts`]({{ site.baseurl}}/2.0/configuration-reference/#store_artifacts) step.
-
-The usage of the [`store_test_results`]({{ site.baseurl}}/2.0/configuration-reference/#store_test_results) key in your config looks like the following:
+Below is an example of using  the [`store_test_results`]({{ site.baseurl}}/2.0/configuration-reference/#store_test_results) key in your `.circleci/config.yml`:
 
 ```sh
 - store_test_results:
     path: test-results
 ```
 
-Where the `path` key is an absolute or relative path to your `working_directory` containing subdirectories of JUnit XML or Cucumber JSON test metadata files, or the path of a single file containing all test results. Make sure that your `path` value is not a hidden folder (example: `.my_hidden_directory` would be an invalid format).
+The `path` key is an absolute or relative path to your `working_directory` containing subdirectories of JUnit XML or Cucumber JSON test metadata files, or the path of a single file containing all test results.
+
+**Note:** Make sure that your `path` value is not a hidden folder. For example, `.my_hidden_directory` would be an invalid format.
+
+## Test Insights
+See the [Test Insights guide]({{site.baseurl}}/2.0/insights-tests/) for information on using the Insights feature to gather information about your tests, including flaky test detection, viewing alist of tests that fail most often, slowest tests and abn overall performance summary.
```

Comment:
Should insights be capitalized?

Reply:
I think this is like the odd one out and is capitalised but I will check :-D

Reply:
Seems to be a yes